feat(gpu): add GPU power save mechanism

zhichao-ambiq opened the pull request:
add GPU power save mechanism

zwang-ambiq commented on src/draw/ambiq/lv_draw_ambiq.c:
```diff
@@ -101,15 +101,6 @@ void lv_draw_ambiq_init(void)
                    draw_ambiq_unit);
 #endif
 
```
I’d suggest adding a guard here to ensure NemaSDK is initialized before calling lv_init. For example:
if (!nema_sdk_is_initialized()) {
    LV_LOG_ERROR(); or LV_ASSERT_MSG();
}
This prevents users from invoking lv_init when NemaSDK hasn’t been set up.

zhichao-ambiq committed:
feat(gpu): add GPU power save mechanism

Implemented a basic GPU power save strategy that powers down or idles
the GPU when no rendering activity
